alma/credentials/merchant_id.py

```diff
@@ -0,0 +1,14 @@
+from . import Credentials
+
+
+class MerchantIdCredentials(Credentials):
+    def __init__(self, merchant_id: str):
+        self.merchant_id = merchant_id
+
+    def configure(self, request):
+        request.headers = {
+            **request.headers,
+            "Authorization": "Alma-Merchant-Auth {merchant_id}".format(
+                merchant_id=self.merchant_id
+            ),
```

Comment:
We should probably use a signed JWK token rather than the plain merchant_id there

Reply:
Probably a discussion for later/elsewhere ... that's how the API is implemented right now 😅